playbooks/documentation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Documentation
+description: How and where to document or share content
+---
+
+# Documentation
+
+As engineers we operate in many parallel forums: GitHub, Jira, Slack, Google docs, e-mail, Notion, and so on. What
+content should go where?
+
+## Documentation or discussion of a specific system
+
+When documentation relates to a particular system, leverage in-repository `README`s or `doc/`s wherever possible.
+This includes when the consumers may not exclusively be engineers.
+
+E.g.:
+
+- How to set-up a local development environment
+- Definitions of domain models
+- API documentation
+
+It _almost_ goes without saying, but other repo-specific interactions such as presenting the rationale for a design
+or offering code-review should leverage the built-in mechanisms of
+[commit messages](/playbooks/engineer-workflow.md#commits),
+[pull requests](/playbooks/engineer-workflow.md#pull-requests), and comments.
+
+## Public versus private
```

Comment:
Super minor: recommend titling “Public versus private content” - will make the document a bit more scannable IMO